units/en/unit3/quiz.mdx

```diff
@@ -0,0 +1,104 @@
+# Quiz [[quiz]]
+
+The best way to learn and [to avoid the illusion of competence](https://www.coursera.org/lecture/learning-how-to-learn/illusions-of-competence-BuFzf) **is to test yourself.** This will help you to find **where you need to reinforce your knowledge**.
+
+### Q1: What are tabular methods?
+
+<details>
+<summary>Solution</summary>
+
+*Tabular methods* are a type of problems in which the state and actions spaces are small enough to approximate value functions to be **represented as arrays and tables**. For instance, **Q-Learning is a tabular method** since we use a table to represent the state,action value pairs.
+
+
+</details>
+
+### Q2: Why we can't use a classical Q-Learning to solve an Atari Game?
+
+<Question
+	choices={[
+		{
+			text: "Atari environments are too fast for Q-Learning",
+			explain: ""
+		},
+		{
+			text: "Atari environments have a big observation space. So creating an updating the Q-Table would not be efficient",
+			explain: "",
+      correct: true
+		}
+	]}
+/>
+
+
+### Q3: Why do we stack four frames together when we use frames as input in Deep Q-Learning?
+
+<details>
+<summary>Solution</summary>
+
+We stack frames together because it helps us **handle the problem of temporal limitation**. Since one frame is not enough to capture temporal information.
```

Comment:
```suggestion
We stack frames together because it helps us **handle the problem of temporal limitation**: one frame is not enough to capture temporal information.
```